### ISSUE TITLE
adminId

### ISSUE THREAD

**Issue by marcinmisiak** [bizley/yii2-podium#94]
Hi,
my user id = 8 (->user->getId())
I've config:
```
'podium' => [
						'class' => 'bizley\podium\Podium',
						'userComponent' => 'user',
						'adminId' => 8,
						'rbacComponent' =>'authManager',
						'dbComponent'=>'dbmoodlewseia',
						'allowedIPs' => ['10.10.10.100'],
						'userPasswordField' => false
				],
```
I'm in  role: podiumAdmin
But do NOT see administration menu in podium 
![obraz](https://cloud.githubusercontent.com/assets/10694258/22817889/49705612-ef6a-11e6-9356-f9bb7d192373.png)
![obraz](https://cloud.githubusercontent.com/assets/10694258/22817906/646a19da-ef6a-11e6-842d-9c714170ab36.png)

**Comment by bizley** [comment]
This is weird, I can see your account is Administrator type (red). Have you seen any error messages during the installation with the configuration above?

**Comment by marcinmisiak** [comment]
No error messages when installing, there's no error in logs,
db logs:
 OK id =8
```
SELECT * FROM `podium_user` WHERE `inherited_id`=8 LIMIT 1

    /var/www/firma/vendor/bizley/podium/src/models/User.php:875
    /var/www/firma/vendor/bizley/podium/src/controllers/BaseController.php:209
```

**Comment by tomaszkane** [comment]
Set breakpoint on `src/views/elements/main/_navbar.php:40` and do some test:
`User::can(Rbac::ROLE_ADMIN);`
`User::can(Rbac::ROLE_USER);`
etc. then you can see what RBAC roles of your user Podium really see.

**Comment by bizley** [comment]
Ok, I've found bug: during installation role is assigned to the value of `adminId` instead of `id` in case of inherited identity. Please try it for quick fix:
- get `id` of your account from `podium_user` table (not `inherited_id`)
- modify your `auth_assignment` table: search for row with `user_id` = 8 and `item_name` = "podiumAdmin" and change `user_id`  to this `id` from `podium_user`
- flush cache if there is one

Let me know if it works.

**Comment by bizley** [comment]
Fixed in `0.7`.
